iothub/service/src/Twin/Models/ClientTwinProperties.cs

```diff
@@ -117,81 +43,12 @@ public int Count
         /// <exception cref="InvalidOperationException">When the specified <paramref name="propertyName"/> does not exist in the collection.</exception>
         public dynamic this[string propertyName]
         {
-            get
-            {
-                if (propertyName == MetadataName)
-                {
-                    return GetMetadata();
-                }
-                else if (propertyName == LastUpdatedName)
-                {
-                    return GetLastUpdatedOnUtc();
-                }
-                else if (propertyName == LastUpdatedVersionName)
-                {
-                    return GetLastUpdatedVersion();
-                }
-                else if (TryGetMemberInternal(propertyName, out dynamic value))
-                {
-                    return value;
-                }
-
-                throw new InvalidOperationException($"Unexpected property name '{propertyName}'.");
-            }
-            set => TrySetMemberInternal(propertyName, value);
-        }
-
-        /// <summary>
-        /// Gets the metadata for this collection.
-        /// </summary>
-        public ClientTwinMetadata GetMetadata()
-        {
-            return new ClientTwinMetadata(GetLastUpdatedOnUtc(), GetLastUpdatedVersion());
-        }
-
-        /// <summary>
-        /// Clears all metadata out of the twin collection as well as the base metadata object.
-        /// </summary>
-        /// <remarks>
-        /// This will remove all metadata from the base metadata object as well as the metadata for the twin collection. 
-        /// This will also clear the underlying metadata object which will affect methods such as
-        /// <see cref="GetMetadata"/> and <see cref="GetLastUpdatedVersion"/>.
-        /// </remarks>
-        public void ClearMetadata()
```

Comment:
Do we still have this?

Reply:
No, I assumed since the metadata is now accessible by properties, no one needs method access to it.

Is there a case for it?